wix-data-core/wix-data.service.json

```diff
@@ -2816,153 +2818,47 @@
           {  } } ],
   "callbacks": [],
   "messages":
-    [ { "name": "WixDataBulkResult",
+    [ { "name": "WixDataBulkRemoveResult",
         "locations":
-          [ { "lineno": 913,
+          [ { "lineno": 448,
+              "filename": "data-api.es6" } ],
+        "docs":
+          { "summary": "An object returned by the `bulkRemove()` function.",
+            "links": [],
+            "examples": [],
+            "extra":
+              {  } },
+        "members":
+          [ { "name": "removed",
+              "type": "number",
+              "doc": "The number of removed items." },
+            { "name": "skipped",
+              "type": "number",
+              "doc": "The number of skipped items." },
+            { "name": "removedItemIds",
+              "type":
+                { "name": "Array",
+                  "typeParams":
+                    [ "string" ] },
+              "doc": "List of IDs of removed items." },
+            { "name": "errors",
+              "type":
+                { "name": "Array",
+                  "typeParams":
+                    [ "Error" ] },
+              "doc": "List of errors." } ],
+        "extra":
+          {  },
+        "labels":
+          [ "new" ] },
+      { "name": "WixDataBulkResult",
+        "locations":
+          [ { "lineno": 923,
               "filename": "data-api.es6" } ],
         "docs":
           { "summary": "An object returned by Wix Data bulk operations.",
             "links": [],
-            "examples":
```

Comment:
For some reason these examples are being removed when they shouldn't. You should check why they are being removed.